doc/03_research/02_perception/first_implementation_plan.md

```diff
@@ -0,0 +1,157 @@
+# First Implementation Plan
+
+This document shows the initial ideas for the implementation of the perception module.
+It includes the various detection and classification modules that are necessary for an efficient and reliable workflow.
+
+---
+## Authors
+
+Marco Riedenauer
+
+## Date
+
+26.11.2022
+
+---
+<!-- TOC -->
+* [First Implementation Plan](#first-implementation-plan)
+  * [Authors](#authors)
+  * [Date](#date)
+  * [Overview](#overview)
+  * [Panoptic Segmentation](#panoptic-segmentation)
+    * [Things and Stuff](#things-and-stuff)
+      * [Things](#things)
+      * [Stuff](#stuff)
+    * [Segmentation Overview](#segmentation-overview)
+    * [Image Panoptic Segmentation](#image-panoptic-segmentation)
+    * [LIDAR Panoptic Segmentation](#lidar-panoptic-segmentation)
+  * [Position Validation](#position-validation)
+  * [Obstacle Detection and Object Classification](#obstacle-detection-and-object-classification)
+  * [Lane Detection](#lane-detection)
+  * [Traffic Light Detection](#traffic-light-detection)
+  * [Traffic Sign Detection](#traffic-sign-detection)
+  * [Prediction](#prediction)
+<!-- TOC -->
+
+---
+## Overview
+
+![](../../00_assets/Umsetzungsplan Perception.jpg)
```

Comment:
GitHub doesn't render this image. Probably the space is the problem. Best would be to remove it and use a english filename.

Reply:
Solved in [cf3092b](https://github.com/ll7/paf22/pull/59/commits/cf3092bacaf21e954fab86c12c076c9a1044ae03)